src/app/views/provisioning/order.html

```diff
@@ -16,6 +16,12 @@ <h1 translate>mno_enterprise.templates.dashboard.provisioning.order.title</h1>
           </div>
         </div>
 
+        <div class="row td-row" ng-if="vm.subscription.product.product_pricings.length === 0">
```

Comment:
You could just go: `ng-if="!vm.subscription.product.product_pricings.length"` as `0` is falsey in JS.

Reply:
Keep consistency with https://github.com/maestrano/mno-enterprise-angular/blob/2.0/src/app/views/provisioning/subscriptions.html#L30